docs/API-V1/2-SINGLE-PORT.md

```diff
@@ -0,0 +1,42 @@
+## Details of a Single Port
+
+#### `- GET /port/{port-name}`
+
+Fetch the general information about a port i.e. the data extracted from the `Portfile`.
+
+Parameters:
+
+| key | type | details | default |
+|------|-----|----|----|
+| field | str, optional | Select only a particular field. If not provided, then all fields are sent.| All |
+
+#### - `GET /port/builds/{port-name}`
+
+Fetch build history of a given port.
+
+Parameters
+
+| key | type | details | default |
+|------|-----|----|----|
+| count | int, optional | Number of builds to be fetched for each builder. | All |
+| builder | str, optional | Specify one more builders, separated by comma (`,`). | All |
+| status | str, optional | Filter by the status of builds. | All |
+
+
+#### - `GET /port/stats/{port-name}`
+
+Fetch installation statistics of a given port.
+
+Parameters
+
+| key | type | details | default
+|------|-----|----|----|
+| days_ago | int, optional | Number of days behind current day, to calculate stats. | 0 |
+| days | int, optional | Number of days behind `days_ago`. Only the submissions made in this period are used to generate the stats. | 30 |
+| criteria | optional | Comma separated list of the criterial of stats. Options: ['total_count', 'req_count', 'os', 'xcode', 'os_arch', 'cxx_stdlib'] | All |
```

Comment:
The first two are clear. For `criteria` it's not entirely clear to me what one would pass to the function and what one would get back. Also `os_arch` may need to be replaced.